### PR TITLE
fix: review header

### PR DESCRIPTION
**Pull request by pregno**
* restore default in theme
* fix header drawer

### CODE REVIEW

File: packages/react/ds/src/header/header.tsx [hunk 1]
```diff
@@ -185,6 +185,11 @@ export function Header({
               id="MobileMenuTrigger"
               type="checkbox"
             />
+            {tools?.menu?.label && (
+              <span className="gi-hidden sm:gi-block gi-text-2md gi-font-bold gi-text-white">
+                {tools?.menu?.label}
```
Comment (hamza14khan) [inline packages/react/ds/src/header/header.tsx:190]: Minor: If `tools?.menu?.label` is true then maybe replace it with `tools.menu.label`

File: packages/html/ds/src/header/components/headerMenu.html [hunk 2]
```diff
@@ -38,11 +41,11 @@
         <span class="gi-text-sm">{{ link.label }}</span>
       </a>
     </li>
-    {% endfor %}
-    {% if props.searchUrl %}
-      <li class="xs:gi-hidden gi-mt-8">
-        {{ HeaderSearchContainer({"searchUrl": props.searchUrl, "dataElement": "search-container-mobile" , "dataTestId": "search-container-mobile"}) }}
-      </li>
+    {% endfor %} {% if props.searchUrl %}
```
Comment (hamza14khan) [inline packages/html/ds/src/header/components/headerMenu.html:44]: question: Is this formatting changes? If yes, then can we please revert it back to the original one as its more readable 😄

**Review by hamza14khan — APPROVED** [review]
Few non blocking comments. Looks good other than that.

File: packages/html/ds/src/header/components/headerMenu.html [hunk 3]
```diff
@@ -1,35 +1,38 @@
-{% from 'header/components/headerSearchContainer.html' import HeaderSearchContainer %}
-{% from 'icon/icon.html' import govieIcon %}
-
-{% macro headerMenu(props) %}
+{% from 'header/components/headerSearchContainer.html' import
+HeaderSearchContainer %} {% from 'icon/icon.html' import govieIcon %} {% macro
+headerMenu(props) %}
```
Comment (BogdanDanilescu) [inline packages/html/ds/src/header/components/headerMenu.html:3]: same here for formatting issues

**Review by BogdanDanilescu — APPROVED** [review]
LGTM with a minor comment about formatting

**Pull request merged**